docs/react-native-gradle-plugin.md

```diff
@@ -150,6 +150,14 @@ The list of flags to pass to `hermesc`. By default is `["-O", "-output-source-ma
 hermesFlags = ["-O", "-output-source-map"]
 ```
 
+### `enableBundleCompression`
+
+Whether the Bundle Asset should be compressed when packaged into a `.apk`, or not.
+
+Disabling compression for the `.bundle` allows it to be directly memory-mapped to RAM, hence improving startup time - at the cost of a larger resulting `.apk` size.
```

Comment:
should we specify that the apk download won't be affected?

Reply:
I've added it